pydocx/export/image_resize.py

```diff
@@ -0,0 +1,165 @@
+# coding: utf-8
+from __future__ import (
+    absolute_import,
+    division,
+    print_function,
+    unicode_literals,
+)
+
+import base64
+from StringIO import StringIO
+from urlparse import urlparse
+import posixpath
+
+import requests
+
+from requests.exceptions import InvalidSchema
+
+from pydocx.util import uri
+from PIL import Image
```

Comment:
This needs to be added to [install_requires](https://github.com/CenterForOpenScience/pydocx/blob/master/setup.py#L54), or otherwise communicated. Ideally, we wouldn't require a working PIL installation if someone didn't want to use this mixin. Thoughts @kylegibson @jlward ?